sourcemash/frontend/static/js/routers/app_router.js

```diff
@@ -6,6 +6,7 @@ Sourcemash.Routers.AppRouter = Backbone.Router.extend({
         "categories/:id": "showCategory",
         "saved": "showSaved",
         "browse": "browseFeeds",
+        "categorizer": "showCategorizer",
```

Comment:
Ahh, I think we should have a way to get to it, but it should be a bit more subtle. Maybe "try out our categorizer" on the splash page just so it's accessible, and then sending the link to Kemeny prize?